torch/csrc/jit/python_compiled_function.cpp

```diff
@@ -224,7 +224,7 @@ struct CompiledFunction {
 
 
 std::ostream& operator<<(std::ostream& out, const CompiledFunction::TraceForKey & trace) {
-  if(!trace.is_ready_) {
+  if(!const_cast<CompiledFunction::TraceForKey&>(trace).ready()) {
```

Comment:
Ugh, do we really have to `const_cast`? We could add a `const` version of `ready()`